src/plugins/discord/sqliteMirror.test.js

```diff
@@ -227,6 +227,43 @@ describe("plugins/discord/sqliteMirror", () => {
       });
     });
 
+    describe("nthMessageIdFromTail", () => {
+      function buildMessages() {
+        const channelId: Snowflake = "1";
+        const authorId: Snowflake = "2";
+
+        const db = new Database(":memory:");
+        const sqliteMirror = new SqliteMirror(db, "0");
+        sqliteMirror.addUser(testUser(authorId));
+        sqliteMirror.addChannel(testChannel(channelId));
+
+        for (let x = 1; x <= 10; x++) {
+          const message = {
+            id: String(x),
+            channelId: channelId,
+            authorId: authorId,
+            timestampMs: x * 2,
+            content: "content",
+            reactionEmoji: [],
+            nonUserAuthor: false,
+            mentions: [],
+          };
+          sqliteMirror.addMessage(message);
+        }
+        return {sqliteMirror, channelId};
+      }
+      it("retrieves nth message from tail", () => {
+        const {sqliteMirror, channelId} = buildMessages();
+        expect(sqliteMirror.nthMessageIdFromTail(channelId, 3)).toBe("7");
+      });
+      it("handles n greater than total message count", () => {
```

Comment:
nit: rephrase to `"returns undefined when n is greater than message count"`

Reply:
Updated